<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageForm</class>
 <widget class="QWidget" name="MessageForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>570</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Новые сообщения</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="nextMessageButton">
     <property name="text">
      <string>Следующее сообщение</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>550</width>
        <height>249</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
